stac-json.test.js

```diff
@@ -0,0 +1,26 @@
+require('jest')
+const fs = require('fs');
+const path = require('path');
+
+const jsonDir = './_output/stac/';
+const files = fs.readdirSync(jsonDir);
```

Comment:
This test crashes if output folder doesn't exists. This use case ( user wants to commit something without building firs) should probably be handled.

Reply:
Tests are now skipped if no _output/stac folder exists.